scripts/build.sh

```diff
@@ -540,10 +540,10 @@ EOF
     sudo chown root:root "$KSU_PRE"
     sudo setfattr -n security.selinux -v "u:object_r:system_file:s0" "$KSU_PRE" || abort
     # Setup init
-#    sudo tee "$SYSTEM_MNT/etc/init/hw/init.rc" <<EOF >/dev/null
-#on property:sys.boot_completed=1
-#    exec - system system -- logwrapper /system/etc/ksuinstall.sh
-#EOF
+    sudo tee "$SYSTEM_MNT/etc/init/hw/init.rc" <<EOF >/dev/null
+on property:sys.boot_completed=1
+    exec u:r:init:s0 -- /system/bin/logwrapper /system/bin/sh /system/etc/ksuinstall.sh
```

Comment:
@YT-Advanced 
After all, this seems to be the reason why it will not start.
Even logcat does not work.

Reply:
Assumed behavior:
```
PS C:\Users\Administrator> adb shell /system/etc/ksuinstall.sh
Checking boot completed
Checking past install
Installing KernelSU APK
Success
Launching KernelSU App
Starting: Intent { cmp=me.weishu.kernelsu/.ui.MainActivity }
Placing completed file
Done!
```

Reply:
ohhh
My mistake completely.

Reply:
Oh, and now it worked

Reply:
In the meantime, I will try the multiple patterns I was going to try earlier.